kolibri/plugins/learn/assets/src/views/CategorySearchModal/CategorySearchOptions.vue

```diff
@@ -123,8 +95,8 @@
   }
 
   export default {
-    name: 'CategorySearchModalOptions',
```

Comment:
Was going to leave a comment here about not doing this because strings - but there are no strings in this component, so I'll leave this comment to preempt any such similar comments!